.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Template for bug reports\
+title: ''
+labels: kind/bug
+assignees: ''
+---
+
+## Version & Operating System
+
+- Dependency Analyzer:
```

Comment:
The repository and tool name might change soon. We'll probably be calling the tool `depstat` 😄

Reply:
Sounds good.